frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ServingRuntimeTemplateSection.tsx

```diff
@@ -192,80 +198,86 @@ const ServingRuntimeTemplateSection: React.FC<ServingRuntimeTemplateSectionProps
-              data-testid="ds-project-image"
-            >
-              <GlobalIcon />
-            </FlexItem>
-            <FlexItem>Global serving runtimes</FlexItem>
-          </Flex>
-        }
-      >
-        {filteredTemplates
-          .filter((template) =>
-            getServingRuntimeDisplayNameFromTemplate(template)
-              .toLocaleLowerCase()
-              .includes(searchServingRuntime.toLocaleLowerCase()),
-          )
-          .map((template, index) => (
-            <MenuItem
-              key={`servingRuntime-${index}`}
-              data-testid={`servingRuntime ${getServingRuntimeNameFromTemplate(template)}`}
-              isSelected={
-                data.servingRuntimeTemplateName === getServingRuntimeNameFromTemplate(template) &&
-                data.scope === SERVING_RUNTIME_SCOPE.Global
-              }
-              onClick={() => {
-                if (
-                  getServingRuntimeNameFromTemplate(template) !== data.servingRuntimeTemplateName ||
-                  data.scope !== SERVING_RUNTIME_SCOPE.Global
-                ) {
-                  setData(
-                    'servingRuntimeTemplateName',
-                    getServingRuntimeNameFromTemplate(template),
-                  );
-                  setData('scope', SERVING_RUNTIME_SCOPE.Global);
+            {filteredScopedTemplates.map((template, index) => (
+              <MenuItem
+                key={`servingRuntime-${index}`}
+                data-testid={`servingRuntime ${getServingRuntimeNameFromTemplate(template)}`}
+                isSelected={
+                  data.servingRuntimeTemplateName === getServingRuntimeNameFromTemplate(template) &&
+                  data.scope === SERVING_RUNTIME_SCOPE.Global
+                }
+                onClick={() => {
+                  if (
+                    getServingRuntimeNameFromTemplate(template) !==
+                      data.servingRuntimeTemplateName ||
+                    data.scope !== SERVING_RUNTIME_SCOPE.Global
+                  ) {
+                    setData(
+                      'servingRuntimeTemplateName',
+                      getServingRuntimeNameFromTemplate(template),
+                    );
+                    setData('scope', SERVING_RUNTIME_SCOPE.Global);
 
-                  setServingRuntimeDisplayName(getServingRuntimeDisplayNameFromTemplate(template));
-                  // Reset model framework selection when changing the template in KServe modal only
-                  if (resetModelFormat) {
-                    resetModelFormat();
+                    setServingRuntimeDisplayName(
+                      getServingRuntimeDisplayNameFromTemplate(template),
+                    );
+                    // Reset model framework selection when changing the template in KServe modal only
+                    if (resetModelFormat) {
+                      resetModelFormat();
+                    }
```

Comment:
The onClick function is getting bigger and it's been used twice in the same file for `SERVING_RUNTIME_SCOPE.Global` and `SERVING_RUNTIME_SCOPE.Project`, it's better if we move this to a helper function.